modules/tinymce/src/themes/silver/main/ts/ui/dialog/IFrame.ts

```diff
@@ -54,9 +54,12 @@ const getDynamicSource = (initialData: Optional<string>, stream: boolean): IFram
 };
 
 const renderIFrame = (spec: IframeSpec, providersBackstage: UiFactoryBackstageProviders, initialData: Optional<string>): SketchSpec => {
+  const isBordered = spec.border;
   const isSandbox = spec.sandboxed;
-  const isTransparent = spec.transparent;
+  const isOpaque = !spec.transparent;
   const baseClass = 'tox-dialog__iframe';
+  const getModClassIf = (condition: boolean, mod: string): string [] =>
+    condition ? [ `${baseClass}--${mod}` ] : [];
```

Comment:
I wonder what it would look like if we just inlined the class arrays into what is now booleans above

e.g.
```
const borderedClass = spec.border ? [ `${baseClass}--bordered` ] : []
// same for opaque
```

and then below
```
classes: [
          baseClass,
          ...opaqueClass,
          ...borderedClass
        ]
```

At some point refactoring code to remove all duplication just makes it harder to follow.

Reply:
I agree, this is a cleaner way to structure the code.